test/features/ReasonsToDonate.ts

```diff
@@ -0,0 +1,69 @@
+import { VueWrapper } from '@vue/test-utils';
+import { expect } from 'vitest';
+import { PageScroller } from '@src/utils/PageScroller/PageScroller';
+import { nextTick } from 'vue';
+import { Tracker } from '@src/tracking/Tracker';
+import { ReasonsToDonateShownEvent } from '@src/tracking/events/ReasonsToDonateShownEvent';
+import { ReasonsToDonateCTAClickedEvent } from '@src/tracking/events/ReasonsToDonateCTAClickedEvent';
+import { ReasonsToDonateItemClickedEvent } from '@src/tracking/events/ReasonsToDonateItemClickedEvent';
+import ReasonsToDonate from '@src/components/ReasonsToDonate/ReasonsToDonate.vue';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const expectContainsReasonsToDonateDialogue = async ( wrapper: VueWrapper<any>, tracker: Tracker ): Promise<any> => {
+	expect( wrapper.findComponent( ReasonsToDonate ).exists() ).toBeTruthy();
+};
+
+const expectScrollsToFormWhenCallToActionIsClicked = async ( wrapper: VueWrapper<any>, pageScroller: PageScroller ): Promise<any> => {
+	await wrapper.find( '.wmde-banner-reasons-to-donate-link' ).trigger( 'click' );
+	await wrapper.find( '.wmde-banner-10-reasons-cta' ).trigger( 'click' );
+	await nextTick();
+
+	expect( pageScroller.scrollIntoView ).toHaveBeenCalledOnce();
+	expect( pageScroller.scrollIntoView ).toHaveBeenCalledWith( '.wmde-banner-form' );
+};
+
+const expectTracksReasonsToDonateShownEvent = async ( wrapper: VueWrapper<any>, tracker: Tracker ): Promise<any> => {
+
+	await wrapper.find( '.wmde-banner-reasons-to-donate-link' ).trigger( 'click' );
+
+	expect( tracker.trackEvent ).toHaveBeenCalledOnce();
+	expect( tracker.trackEvent ).toHaveBeenCalledWith( new ReasonsToDonateShownEvent() );
+};
+
+const expectTracksReasonsToDonateCTAClickedEvent = async ( wrapper: VueWrapper<any>, tracker: Tracker ): Promise<any> => {
+
+	await wrapper.find( '.wmde-banner-reasons-to-donate-link' ).trigger( 'click' );
+	await nextTick();
+
+	console.log( wrapper.html() );
+
+	await wrapper.find( '.wmde-banner-10-reasons-cta' ).trigger( 'click' );
+	await nextTick();
+
+	expect( tracker.trackEvent ).toHaveBeenCalledTimes( 2 );
+
+	expect( tracker.trackEvent ).toBeCalledWith( new ReasonsToDonateCTAClickedEvent() );
+};
+
+const expectTracksReasonsToDonateItemClickedEvent = async ( wrapper: VueWrapper<any>, tracker: Tracker ): Promise<any> => {
+
+	await wrapper.find( '.wmde-banner-reasons-to-donate-link' ).trigger( 'click' );
+	await nextTick();
+
+	await wrapper.find( '.wmde-banner-10-reasons-accordion-item' ).trigger( 'click' );
```

Comment:
There's an example of what you need to do in the `ReasonsToDonate.spec.ts` tests
```suggestion
	wait wrapper.find( '.wmde-banner-10-reasons-accordion-item:nth-child(1) .wmde-banner-10-reasons-accordion-title' ).trigger( 'click' );
	await wrapper.find( '.wmde-banner-10-reasons-accordion-item:nth-child(8) .wmde-banner-10-reasons-accordion-title' ).trigger( 'click' );
```

Then check the nth-child number is passed to the event as it's close choice.